app/models/entities/sub_entities/lead.rb

```diff
@@ -0,0 +1,31 @@
+class Entities::SubEntities::Lead < Maestrano::Connector::Rails::SubEntityBase
+
+  def self.external?
+    true
+  end
+
+  def self.entity_name
+    'Lead'
+  end
+
+  def self.mapper_classes
+    {
+      'Person' => Entities::SubEntities::PersonMapper,
+      'Organization' => Entities::SubEntities::OrganizationMapper
+    }
+  end
+
+  def self.references
+    {
+      'Person' => %w(organization_id, assignee_id)
+    }
+  end
+
+  def self.object_name_from_external_entity_hash(entity)
+    if entity['last_name']
```

Comment:
In Base the last_name or organization_name are mandatory, but you cannot have only the first name